api/pkg/cli/hub/get_resource.go

```diff
@@ -162,8 +190,17 @@ func (t *tektonHubclient) GetResourcesList(so SearchOption) ([]string, error) {
 }
 
 func (a *artifactHubClient) GetResourceVersionslist(r ResourceOption) ([]string, error) {
-	// Todo: implement GetResourceVersionslist for Artifact Hub
-	return []string{}, nil
+	data, _, err := a.Get(fmt.Sprintf("%s-%s/%s/%s", artifactHubCatInfoEndpoint, r.Kind, r.Catalog, r.Name))
```

Comment:
Not really related to this patch, but should we add a comment above the function just for info, regarding this function is for getting the resource version list for Artifact Hub